scripts/java-api-check.json

```diff
@@ -10,7 +10,56 @@
       "filter": {
         "classes": {
           "regex": true,
-          "include": ["com\\.mapbox\\.maps\\..*"]
+          "include": ["com\\.mapbox\\.maps\\..*"],
```

Comment:
@ank27 is it left here intentionally? I guess better to deliver it in a separate PR

Reply:
yeah, removing this commit.